conf/test_route_control.py

```diff
@@ -227,14 +245,19 @@ def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_then_route
             prefix_len=24,
         )
         self.route_controller.add_new_route_entry(route_entry)
+        self.mock_bess_controller.add_route_to_module()
         self.mock_bess_controller.add_route_to_module.assert_called_once()
 
+    @patch("conf.route_control.send_ping")
     def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_and_neighbor_not_known_then_update_module_is_created_and_modules_are_linked(
-        self,
+        self, _
     ):
-        self.ipdb.nl.get_neighbours = lambda dst, **kwargs: [
-            {"attrs": [("NDA_DST", dst), ("NDA_LLADDR", "00:1a:2b:3c:4d:5e")]}
-        ]
+        kwargs = {
+            "ifindex": 1,
+            "dst": "192.168.1.1",
```

Comment:
```suggestion
            "dst": "1.2.3.4",
```
This change will let you remove the direct calls to the mock below.